### PR TITLE
Fix a small failing test

### PR DESCRIPTION
**Pull request by chaimain**

### CODE REVIEW

**Comment by rlopezcoto** [comment]
tests are not gonna pass unless all of them are fixed... maybe @maxnoe or @FrancaCassol can push the fixes mentioned in:
https://github.com/cta-observatory/cta-lstchain/pull/1054#issuecomment-1554079544 to this branch

**Comment by chaimain** [comment]
> tests are not gonna pass unless all of them are fixed... maybe @maxnoe or @FrancaCassol can push the fixes mentioned in: [#1054 (comment)](https://github.com/cta-observatory/cta-lstchain/pull/1054#issuecomment-1554079544) to this branch

True, but if you merge this, we will have 1 less test to worry about in the main `ctapipe_0.17` branch. That is in case @FrancaCassol or @maxnoe prefer to resolve that test in a different PR.

**Comment by rlopezcoto** [comment]
it *cannot* be merged unless test pass (or we bypass branch protections). I would rather solve it for good as I suggested instead of deactivating branch protections

File: lstchain/reco/tests/test_utils.py [hunk 1]
```diff
@@ -209,21 +221,28 @@ def test_get_geomagnetic_delta():
...
-    src_r_min = srcdep_config['train_gamma_src_r_deg'][0]
-    src_r_max = srcdep_config['train_gamma_src_r_deg'][1]
+
+    src_r_min = srcdep_config["train_gamma_src_r_deg"][0]
+    src_r_max = srcdep_config["train_gamma_src_r_deg"][1]
+
     params = apply_src_r_cut(params, src_r_min, src_r_max)
-    assert (params.event_id.values == np.array([998705, 1680619])).all()
+
+    assert (params.event_id.values == np.arange(100, 110, 1)).all()
```
Comment (chaimain) [inline lstchain/reco/tests/test_utils.py:248]: This is the only difference in the actual code. Sorry for mixing in with the formatting edit.

**Comment by chaimain** [comment]
Closing this here for a cleaner PR in #1107

**Pull request closed without merging**